app/models/repo.rb

```diff
@@ -2,7 +2,8 @@
 
 class Repo < ActiveRecord::Base
   validate :github_url_exists, on: :create
-  validates :name, uniqueness: {scope: :user_name, case_sensitive: false }
+
+  # Now done at the DB level # validates :name, uniqueness: {scope: :user_name, case_sensitive: false }
```

Comment:
Space after opening `{` :) (also dead code in comments, and doesn't need to have a comment if there is a spec for it)